app/requirements.txt

```diff
@@ -10,4 +10,6 @@ flask-compress==1.4.0
 selenium==3.5.0
 nose==1.3.7
 pypdf2==1.26.0
+defusedxml==0.5.0
+cairocffi==0.6.0

```

Comment:
@gabru-md You don't need to add `cairocffi` package since it will be installed as a dependency while installing `cairosvg`. You just need the correct python dev package installed for your OS. We can add this in readme perhaps for Troubleshooting.